cli/tools/test/mod.rs

```diff
@@ -1701,7 +1701,7 @@ pub async fn run_tests(
       flags,
       vec![coverage.clone()],
       vec![],
-      vec![],
+      vec!["^file:".to_string()],
```

Comment:
This argument is mainly for handling exclude pattern given by the user. Can you instead modify `filter_coverages` function to exclude those files?

Reply:
What would be the best approach for this ?

Do I add something like `!is_data` or should I directly append the `^data:` regex in the `is_excluded` vector ?
https://github.com/denoland/deno/blob/5bd9a337c1da8c7cad22c7760c02f80314021eea/cli/tools/coverage/mod.rs#L589-L592